web/src/app/schedules/ScheduleShiftList.js

```diff
@@ -328,12 +342,21 @@ export default class ScheduleShiftList extends React.PureComponent {
               <ScheduleTZFilter scheduleID={this.props.scheduleID} />
             </Grid>
             <Grid item xs={12}>
-              <DatePicker
+              <KeyboardDatePicker
+                data-cy='start-date'
                 className={this.props.classes.datePicker}
                 disabled={this.props.activeOnly}
                 label='Start Date'
-                value={DateTime.fromISO(this.props.start, { zone })}
-                onChange={e => this.props.handleSetStart(e.toISO())}
+                format='MM/dd/yyyy'
+                mask='__/__/____'
+                placeholder={DateTime.local().toFormat('MM/dd/yyyy')}
+                value={this.state.startDate}
+                onChange={date => {
+                  if (date && date.isValid) {
+                    this.props.handleSetStart(date.toISO())
```

Comment:
Should this be DateTime?

Reply:
This is the ISO timestamp stored in the URL, if the date typed in is valid.